### PR TITLE
Add SanitizedLogger and use it in gramine

### PR DESCRIPTION
**Pull request by kvinwang**

### CODE REVIEW

File: crates/phactory/src/contracts/support.rs [hunk 1]
```diff
@@ -190,7 +190,7 @@ impl FatContract {
         phala_mq::select! {
             next_cmd = self.cmd_rcv_mq => match next_cmd {
                 Ok((_, cmd, origin)) => {
-                    info!(target: "contract", "Contract {:?} handling command", self.id());
+                    info!("Contract {:?} handling command", self.id());
```
Comment (h4x3rotab) [inline crates/phactory/src/contracts/support.rs:193]: What will happen if the target is not specified?

File: standalone/pruntime/src/logger.rs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+fn target_allowed(target: &str) -> bool {
+    use MatchMode::*;
+    enum MatchMode {
+        Prefix,
+        Eq,
+    }
+
+    // Keep more frequantly targets in the front
```
Comment (h4x3rotab) [inline standalone/pruntime/src/logger.rs:47]:
```suggestion
    // Keep more frequently targets in the front
```

**Review by h4x3rotab — COMMENTED** [review]
LGTM

File: crates/phactory/src/contracts/support.rs (hunk 1 above)
Comment (kvinwang) [reply crates/phactory/src/contracts/support.rs:193]: It would use the module path as the target. `phactory::contracts` for example.

Comment (h4x3rotab) [reply crates/phactory/src/contracts/support.rs:193]: Sounds good

**Pull request merged**